pgjdbc/src/main/java/org/postgresql/PGConnection.java

```diff
@@ -379,4 +381,20 @@ default void alterUserPassword(String user, char[] newPassword, @Nullable String
    * @return state of adaptive fetch (turned on or off)
    */
   boolean getAdaptiveFetch();
+
+  void setPlaceholderStyle(PlaceholderStyle placeholderStyle);
+
+  PlaceholderStyle getPlaceholderStyle();
+
+  PreparedStatement prepareStatement(String sql, PlaceholderStyle placeholderStyle) throws SQLException;
+
+  PreparedStatement prepareStatement(String sql, int autoGeneratedKeys, PlaceholderStyle placeholderStyle) throws SQLException;
+
+  PreparedStatement prepareStatement(String sql, int @Nullable [] columnIndexes, PlaceholderStyle placeholderStyle) throws SQLException;
+
+  PreparedStatement prepareStatement(String sql, String @Nullable[] columnNames, PlaceholderStyle placeholderStyle) throws SQLException;
+
+  PreparedStatement prepareStatement(String sql, int resultSetType, int resultSetConcurrency, PlaceholderStyle placeholderStyle) throws SQLException;
+
+  PreparedStatement prepareStatement(String sql, int resultSetType, int resultSetConcurrency, int resultSetHoldability, PlaceholderStyle placeholderStyle) throws SQLException;
```

Comment:
Do we want duplicating all the prepare methods? Is it a shortcut for `setPlacehoderStyle(...); prepareStatement(...)`?

If that is a shortcut, I would avoid adding new prepareStatement, otherwise we could get quite a few of them quite fast